packages/fluentui/react-northstar/src/utils/positioner/Popper.tsx

```diff
@@ -226,6 +280,26 @@ export const Popper: React.FunctionComponent<PopperProps> = props => {
     return destroyInstance;
   }, [createInstance]);
 
+  if (process.env.NODE_ENV !== 'production') {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    React.useEffect(() => {
+      if (contentRef.current) {
+        const contentNode = contentRef.current;
+        const reactInstanceKey = getReactInstanceKey(contentNode);
+        const treeWalker = contentNode.ownerDocument?.createTreeWalker(contentNode, NodeFilter.SHOW_ELEMENT, {
+          acceptNode: node =>
+            isAutofocusEnabled(node, reactInstanceKey) ? NodeFilter.FILTER_ACCEPT : NodeFilter.FILTER_SKIP,
```

Comment:
Once `hasAutofocusProp` will accept a single param we can move this condition inside of it

```suggestion
          acceptNode: hasAutofocusProp,
```